src/coreclr/src/jit/importer.cpp

```diff
@@ -20669,7 +20638,7 @@ void Compiler::impDevirtualizeCall(GenTreeCall*            call,
     // final method that objClass inherits.
     CORINFO_CLASS_HANDLE derivedClass = info.compCompHnd->getMethodClass(derivedMethod);
 
-    // Need to update call info too. This is fragile
+    // Need to update call info too. This is fragile and suboptimal
```

Comment:
Is there some follow-up we should consider here?

Reply:
Opened https://github.com/dotnet/runtime/issues/38477.

Reply:
It is pre-existing issue - independent on the changes in this PR.